docs/files.md

```diff
@@ -93,3 +93,132 @@ The following example shows `graph.json` file for the top-level dependency `whee
   }
 }
 ```
+
+## Output Directories
+
+During the wheel building process, fromager generates multiple output directories namely `sdists-repo`, `wheels-repo` and `work-dir`. These directories contain important information related to the wheel build.
+
+### sdist-repo
+
+This directory contains the source distributions for the package and its dependencies that we are building. The directory structure of `sdists-repo` is as follows
+
+```
+sdists-repo
+├── builds
+└── downloads
+```
+
+The `builds` and `downloads` sub-directories contain the source distributions that are built and downloaded respectively in `tar.gz` format. For example, the `sdists-repo` for `stevedore` package looks as follows:
+
+```
+sdists-repo
+├── builds
+│   ├── pbr-6.1.0.tar.gz
+│   ├── setuptools-75.1.0.tar.gz
+│   └── stevedore-5.3.0.tar.gz
+└── downloads
+    ├── pbr-6.1.0.tar.gz
+    ├── setuptools-75.1.0.tar.gz
+    └── stevedore-5.3.0.tar.gz
+
+```
+
+We can see source distributions for `pbr` and `setuptools` since these are dependencies of `stevedore`.
+
+### wheels-repo
+
+This directory contains the wheels for a package and its dependencies that are built by fromager, used as prebuilt and the ones that are downloaded from indices. The directory structure of `wheels-repo` is as follows
+
+```
+wheels-repo
+├── build
+├── downloads
+├── prebuilt
+└── simple
+```
+
+* The `build` sub-directoy contains the configuration information related to the wheel build
```

Comment:
The build directory holds temporary builds. We use it as the output directory when building the wheel because we can't predict the filename, and so using an empty directory with a name we know gives us a way to find the file and move it into the `downloads` directory after it's built.

Explaining that is highlighting for me that we may want to change some of these names.